src/index.js

```diff
@@ -444,20 +444,21 @@ export default {
 
   async buildGetCredDefRequest(submitterDid: Did, id: string): Promise<LedgerRequestResult> {
     if (Platform.OS === 'ios') {
-      return IndySdk.buildGetCredDefRequest(submitterDid, id)
+      return JSON.parse(await IndySdk.buildGetCredDefRequest(submitterDid, id));
```

Comment:
This now matches the signature of the non ios statement so I think the `if (Platform.OS === 'ios')` could be deleted right?